src/main/java/org/ays/auth/security/AysAuthenticationEntryPoint.java

```diff
@@ -22,13 +22,14 @@
 @Component
 public class AysAuthenticationEntryPoint implements AuthenticationEntryPoint {
 
-    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private final ObjectMapper objectMapper;
```

Comment:
Bu yaptığınız değişiklik ile birlikte sınıfın her instancesi için yeni bir objectMapper oluşturulur. Bunu istemediğimiz için static tanımlıyoruz, kaldı ki ObjectMapper thread safe bir yapıda.